Octokit.Tests/Clients/MiscellaneousClientTests.cs

```diff
@@ -31,7 +31,27 @@ public class TheRenderRawMarkdownMethod
                     "text/plain");
             }
         }
+        public class TheRenderArbitrryMarkdownMethod
+        {
+            [Fact]
+            public async Task RequestsTheEmojiEndpoint()
```

Comment:
This test description should mention that it's interested in the markdown endpoint.